socrates/lib/members/views/userinfoform.pug

```diff
@@ -29,6 +29,9 @@ mixin memberinfoform(member, subscriber, newmember_text, allCountries)
           +text('lastname', 'Last name', member.lastname())
         .col-sm-12
           +singleselect('country', 'Country of Residence', subscriber ? subscriber.country() : null, 'Select your country', allCountries)
+        .col-sm-12
+          +text('pronoun', 'Preferred pronoun (will appear on your name tag)', subscriber && (typeof subscriber.addon === 'function') ? subscriber.addon().pronoun():'')
```

Comment:
Also Du meinst, Du baust etwas in Produktionscode ein, was da überhaupt keinen Sinn macht, damit irgendwelche Tests durchlaufen?